src/tools/pontoon/adding_new_terminology.md

```diff
@@ -1,25 +1,25 @@
-# Adding new terminology on Pontoon
+# Adding new terms to terminology on Pontoon
 
-To add new terminology to Pontoon access Django’s admin interface at `https://pontoon.mozilla.org/a/` (note that this is not the usual admin interface), then follow the steps below.
-
-## Add the new term
+To add new terms to Pontoon, access Django’s admin interface at `https://pontoon.mozilla.org/a/` (note that this is not the usual admin interface), then follow the steps below.
 
 Find and click `Terms` on the navigation pane on the left. Check on the following page to make sure a term does not already exist by searching in the search field near the top.
 
-To add a new term, click `Add Term +`. The following page will have the following fields:
+To add a new term, click `ADD TERM +`. The next page will have the following fields:
 
 * Text: Term you wish to register.
-* Entity: *Ignore*
+* Entity: *Leave empty*.
 * Part of speech: Select the part of speech that applies to your term. In some cases the same string can be registered twice with different parts of speech, e.g. bookmark as a noun (“open your bookmarks”) or as a verb (“bookmark this website”).
-* Definition: Definition of the meaning of the term, or explanation of what the term is.
+* Definition: Meaning of the term, or explanation of what the term is.
 * Usage: Example usage of the term.
-* Notes: Any other notes or context that could be pertinent to localizers.
-* Case sensitive: Flag if the term should only match when case matches.
-* Exact match: Flag if the term should only match on an exact match. (Unflagged allows fuzzy matching.)
-* Do not translate: Flag if the term should not be translated (example brand names like Firefox).
-* Forbidden: Flag if this term is forbidden from usage.
-* Created by: *Ignore*
+* Notes: Any other notes or context that could be relevant.
+* Case sensitive: Select if the term should only match when case matches.
+* Exact match: Select if the term should only match a perfect match. If not selected, fuzzy matching will be used.
```

Comment:
@mathjazz 

Flagged you for review, but I'm mostly interested in this bit. I could find a [definition](https://github.com/mozilla/pontoon/blob/17138c608c992358eddd24144a2a049960f794bf/pontoon/terminology/models.py#L83) in the code, but nothing that uses that.

Reply:
That's correct. The flag is only in because it was in the spreadsheet.

Should we remove it or use it somehow?

Reply:
Unless we have plans to implement it — and I'm not exactly sure how the algorithm would look like in that case — I think it would make sense to remove it.

Reply:
Filed https://github.com/mozilla/pontoon/issues/2629.

Reply:
Thanks, I'm going to remove this bullet point here then.